app/src/main/java/skills/future/planer/ui/day/DayViewModel.java

```diff
@@ -1,12 +1,101 @@
 package skills.future.planer.ui.day;
 
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.ViewModel;
+import androidx.viewpager.widget.ViewPager;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.prolificinteractive.materialcalendarview.CalendarDay;
+import com.prolificinteractive.materialcalendarview.MaterialCalendarView;
+
+import lombok.Getter;
+import skills.future.planer.ui.day.views.daylist.DayTaskListViewModel;
+import skills.future.planer.ui.day.views.matrix.MatrixModelView;
 
+@Getter
 public class DayViewModel extends ViewModel {
 
```

Comment:
To nie powinno rozszerzać po AndroidViewModel

Reply:
Nie, jak tworzysz domyślnie Fragment  + ViewModel to generuje po tym rozszerzeniu